dnf/db/group.py

```diff
@@ -321,7 +321,8 @@ def _populate_rpm_ts(self, ts):
                 modular_problems += self._test_fail_safe(hdr, tsi.pkg)
                 ts.addReinstall(hdr, tsi)
             elif tsi.action == libdnf.transaction.TransactionItemAction_REINSTALLED:
-                pass
+                # Resolves issue when multiple packages with same NEVRA installed
```

Comment:
I'm not sure these are the packages with the same NEVRA? Aren't they those with different version?

Anyway, I'd phrase it like (feel free to adjust it):
>     # When multiple packages with the same NEVRA are marked as installed, remove <WHICH_ONES>

Main point for me is to say _marked as installed_ and not _installed_, as I find it really confusing, the packages aren't really installed on the system at the same time... Cheers :slightly_smiling_face:

Reply:
The comment was replaced by:

```
# Required when multiple packages with the same NEVRA marked as installed
```